features/raadpleeg-verblijfplaats-met-periode/gba/overzicht-gba.feature

```diff
@@ -0,0 +1,255 @@
+    | 0800010000000001                 | Laan   | 20100818 |          |
+
+    Voorbeelden:
+    | datum van  | scenario                                                                              |
+    | 2010-09-01 | De gevraagde periode ligt in de adreshoudingperiode op een adresseerbaar object       |
+    | 2010-08-01 | De gevraagde periode ligt deels in de adreshoudingperiode op een adresseerbaar object |
+
+  Scenario: De gevraagde periode ligt in de adreshoudingperiode op een adresseerbaar object (wel vorige en geen volgende adreshoudingperiode)
+    Gegeven de persoon met burgerservicenummer '000000024' is ingeschreven op adres 'A1' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20100818                           |
+    En de persoon is vervolgens ingeschreven op adres 'A2' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20160526                           |
+    Als gba verblijfplaatsen wordt gezocht met met de volgende parameters
+    | naam                | waarde              |
+    | type                | RaadpleegMetPeriode |
+    | burgerservicenummer | 000000024           |
+    | datumVan            | 2016-09-01          |
+    | datumTot            | 2020-01-01          |
+    | fields              | verblijfplaats      |
+    Dan heeft de response verblijfplaatsen met de volgende gegevens
+    | adresseerbaarObjectIdentificatie | straat      | datumVan | datumTot |
+    | 0800010000000002                 | Luttestraat | 20160526 |          |
+
+  Scenario: De gevraagde periode ligt in de adreshoudingperiode op een adresseerbaar object (geen vorige en wel volgende adreshoudingperiode)
+    Gegeven de persoon met burgerservicenummer '000000024' is ingeschreven op adres 'A1' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20100818                           |
+    En de persoon is vervolgens ingeschreven op adres 'A2' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20160526                           |
+    Als gba verblijfplaatsen wordt gezocht met met de volgende parameters
+    | naam                | waarde              |
+    | type                | RaadpleegMetPeriode |
+    | burgerservicenummer | 000000024           |
+    | datumVan            | 2011-01-01          |
+    | datumTot            | 2016-01-01          |
+    | fields              | verblijfplaats      |
+    Dan heeft de response verblijfplaatsen met de volgende gegevens
+    | adresseerbaarObjectIdentificatie | straat | datumVan | datumTot |
+    | 0800010000000001                 | Laan   | 20100818 | 20160526 |
```

Comment:
je vult hier de datumTot met de datum van een volgende die buiten de gevraagde periode ligt. Zouden we hier niet, net als we bij bewoning doen, alleen informatie moeten leveren die binnen de gevraagde periode ligt?
zelfde geldt voor datumVan die voor de gevraagde periode ligt.

Dus datumVan >= periode.datumVan en datumTot <= periode.datumTot

```
    | adresseerbaarObjectIdentificatie | straat | datumVan | datumTot |
    | 0800010000000001                 | Laan   | 20110101 | 20160101 |
```

Of doen we dat in de proxy pas?

Reply:
> je vult hier de datumTot met de datum van een volgende die buiten de gevraagde periode ligt. Zouden we hier niet, net als we bij bewoning doen, alleen informatie moeten leveren die binnen de gevraagde periode ligt? zelfde geldt voor datumVan die voor de gevraagde periode ligt.
> 
> Dus datumVan >= periode.datumVan en datumTot <= periode.datumTot
> 
> ```
>     | adresseerbaarObjectIdentificatie | straat | datumVan | datumTot |
>     | 0800010000000001                 | Laan   | 20110101 | 20160101 |
> ```
> 
> Of doen we dat in de proxy pas?

Ik denk dat als je dit doet dat je een vertekend beeld geeft van de verblijfplaatshistorie van een persoon:
- als datumVan of datumTot van de gevraagde periode in een onzekerheidsperiode valt, dan weet je niet dat datum aanvang adreshouding geheel/deels onbekend is
- je weet niet of de eerste verblijfplaats die wordt geleverd de huidige verblijfplaats is van de persoon of niet. Zelfs als je met datumTot vandaag vraagt, krijgt je een datumTot met waarde

Verder zou je de resource geen verblijfplaats moeten noemen omdat datumVan conflicteert met datumVan van de verblijfplaats resource in de Bevragen API. In de Bevragen API is datumVan gelijk aan datum aanvang adreshouding/adres buitenland en in deze API is datumVan óf gelijk aan datum aanvang adreshouding/adres buitenland óf gelijk aan datumVan van de gevraagde periode

Reply:
afgesproken dat de GBA API datums levert zoals die in de registratie staan. Dus datum aanvang en datum aanvang volgende wordt niet aangepast naar de gevraagde periode.